src/main/java/com/linkedin/kmf/services/ConsumeService.java

```diff
@@ -118,7 +118,7 @@ public void run() {
           LOG.error(_name + "/ConsumeService failed", e);
         }
       }
-    }, _name + " consume-service");
+    }, _name + " consume-service-thread");
```

Comment:
I think I had the string "thread" in the thread names at some point and we decided that it can make the thread names too long.